renderer/index.js

```diff
@@ -457,11 +445,11 @@ function onOpen (files) {
 
   // .torrent file = start downloading the torrent
   files.filter(isTorrent).forEach(function (torrentFile) {
-    addTorrent(torrentFile)
+    addTorrent(torrentFile.path)
   })
 
   // everything else = seed these files
-  showCreateTorrentModal(files.filter(isNotTorrent))
+  createTorrentFromFileObjects(files)
```

Comment:
Oh, I see it below in `createTorrentFromFileObjects`. Moving it back here to avoid confusion.